oam/install_scripts/mariadb-columnstore-stop.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# This script allows to gracefully shut down MCS
+
+/bin/systemctl stop mcs-dmlproc
+/bin/systemctl stop mcs-ddlproc
+/bin/systemctl stop mcs-primproc
+/bin/systemctl stop mcs-writeengineserver
+/bin/systemctl stop mcs-storagemanager
+/bin/systemctl stop mcs-exemgr
```

Comment:
Same as in mariadb-columnstore-start

Reply:
Ditto, but reverse order.

Reply:
Despite the fact we can rely on systemd dependenices here I'll change the order to make it explicit for an external viewer.